components/resume/resume.tsx

```diff
@@ -1,54 +1,34 @@
 import React, { FC } from 'react';
-import Company from './company/company';
+import Job from './job/job';
+import Skills from './skills';
+import { Basics, Job as IJob } from './types';
```

Comment:
Not a huge fan of this `IJob` renaming, did it due to conflicts with a component and type having the same name. Other workarounds would be to not destruct the props passed into the component or come up with a different naming scheme. Not a fan of the former and not sure what I'd want the latter to be. So going with the quick IJob rename here.